community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>consider giving a default name to the endpoint display name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>